lib/unexpectedMitm.js

```diff
@@ -730,6 +730,46 @@ module.exports = {
                     return recordedExchanges;
                 });
             })
+            .addAssertion('<any> with http mocked out [and verified] [returning the file] <string> <assertion>', function (expect, subject, filePathWithoutExt) {
+                expect.errorMode = 'nested';
+                var testFile = filePathWithoutExt + '.js';
+
+                if (process.env.UNEXPECTED_MITM_WRITE === 'true') {
```

Comment:
First I tried `UNEXPECTED_MITM_WRITE=yes` and was puzzled why it didn't work. Consider supporting any value like https://github.com/unexpectedjs/unexpected/blob/a2f14d4fe03a6eccb0f621411ff581c5b6ad78f6/lib/useFullStackTrace.js#L8